doc/iojs.1

```diff
@@ -101,16 +101,10 @@ If set to 1 then colors will not be used in the REPL.
         type: bool  default: false
   --harmony_numeric_literals (enable "harmony numeric literals")
         type: bool  default: true
-  --harmony_strings (enable "harmony string methods")
-        type: bool  default: true
-  --harmony_scoping (enable "harmony block scoping")
-        type: bool  default: true
   --harmony_classes (enable "harmony classes (implies block scoping & object literal extension)")
```

Comment:
Does the flag enable it for sloppy mode? Currently classes are only available for unflagged use in strict mode.